lib/menu.rb

```diff
@@ -0,0 +1,6 @@
+class Menu
```

Comment:
Love that the menu is a separate class from the take-away. This could be built on to add different styles of menu for different new restaurant instances. nice!